x-pack/plugins/siem/public/components/page/network/domains_table/columns.tsx

```diff
@@ -146,40 +149,24 @@ export const getDomainsColumns = (
     },
   },
   {
-    name: i18n.FIRST_SEEN,
+    name: (
+      <EuiToolTip content={i18n.FIRST_LAST_SEEN_TOOLTIP}>
+        <span>
+          {i18n.LAST_SEEN}
+          <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+        </span>
+      </EuiToolTip>
+    ),
     truncateText: false,
     hideForMobile: false,
     render: ({ node }) => {
-      const domainNameAttr = `${flowTarget}.domainName`;
-      const domainName = getOr(null, domainNameAttr, node);
-      if (domainName != null) {
-        return (
-          <FirstLastSeenDomain
-            ip={ip}
-            domainName={domainName}
-            flowTarget={flowTarget}
-            type="first-seen"
-          />
-        );
-      }
-      return getEmptyTagValue();
-    },
-  },
-  {
-    name: i18n.LAST_SEEN,
-    truncateText: false,
-    hideForMobile: false,
-    render: ({ node }) => {
-      const domainNameAttr = `${flowTarget}.domainName`;
-      const domainName = getOr(null, domainNameAttr, node);
-      if (domainName != null) {
+      const lastSeenAttr = `${flowTarget}.lastSeen`;
```

Comment:
Looking at `render: ({ node })`, because we are using `ValueOf<DomainsNode>` and that is bringing in the type of `any` from Date time, this `render` is showing node as any any:

<img width="413" alt="Screen Shot 2019-04-17 at 1 36 52 PM" src="https://user-images.githubusercontent.com/1151048/56315855-10c48480-6116-11e9-8308-1992b23a6cb5.png">

I would suggest we change the types to be the correct ones without the `any` so when we move DateTime over we will be A ok.

Reply:
Good catch! Looks like the root type `DomainsNode` was used instead of column types. I've updated the column types to be:

``` typescript
Array<Columns<ValueOf<DomainsItem> | ValueOf<DomainsNetworkField>>>
```

This is a great fix and gets us on our way to removing the `Date` `any` issue you found yesterday in dealing with the gql-type-gen. 🎉